src/test/java/uk/gov/companieshouse/filinghistory/api/controller/AnnotationTransactionIT.java

```diff
@@ -528,7 +585,7 @@ void shouldThrowConflictExceptionWhenUpdatingChildAnnotationWithStaleDeltaAtAndR
         // Assert existing doc is unchanged
         assertEquals(existingDocument, actualDocument);
 
-        verifyNoMoreInteractions(instantSupplier);
+        verify(instantSupplier, times(1)).get();
```

Comment:
can be ```verify(instantSupplier).get();``` as the default number of invocations is 1

Reply:
all instances corrected.